### ISSUE TITLE
Add Footer styling / styled components

### ISSUE THREAD

**Issue by riddlew** [CommunityFocus/CommunityFocus#49]
## What
Create the styled component(s) and styling for the footer component (#47) and its child components (MembersUsingMessage). This issue depends on #47 being finished, so this cannot be done until that issue is complete.

Please reference the Figma design at https://www.figma.com/file/aOxY6JeMesN7fq72tZsbD9/Untitled?type=design&t=ojy9KXiE1A3V8cHZ-0 for an idea of how this component will look (design subject to change). This component is located at bottom top of the screen.

![Footer](https://github.com/CommunityFocus/CommunityFocus/assets/6940264/5ecf951c-3716-41f6-9f42-7b049e18c8fa)

## Details
- Add styling and styled component(s) to the Footer component and child components (MembersUsingMessage).
- No testing required as of right now, this will be handled in a separate issue.

**Comment by Natashka333** [comment]
@nmpereira I will take the issue
